docs/genai/data-mgmt/vector-databases.md

```diff
@@ -50,6 +50,228 @@ q_context = " ".join([f"#{str(i)}" for i in results["documents"][0]])
 prompt_template = f"Relevant context: {q_context}\n\n The user's question: {question}"
 ```
 
+## Vector database ingestion and inference pipeline
+The following example runs an mlrun workflow that creates and ingests data into a Milvus vector DB, then it deploys a nuclio function that enables to query the vector DB.
```

Comment:
The following example runs an MLRun workflow that creates and ingests data into a Milvus vector DB, then it deploys a Nuclio function to query the vector DB.